allennlp/tests/training/metrics/auc_test.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=no-self-use,invalid-name,protected-access
+import torch
+from sklearn import metrics
+from numpy.testing import assert_almost_equal
+import pytest
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.training.metrics import Auc
+from allennlp.common.checks import ConfigurationError
+
+
+class AucTest(AllenNlpTestCase):
```

Comment:
Could you add a test that checks that the AUC metric doesn't break if you call `get_metrics` without ever having called the metric itself?